org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/window/AboutWindow.java

```diff
@@ -367,7 +367,7 @@ protected Tabpanel createInfo() {
 		LayoutUtils.addSclass("about-info-panel", div);
 		div.setParent(tabPanel);
 		ZKUpdateUtil.setHeight(div, "100%");
-		div.setStyle("overflow: auto;");
+		div.setStyle("overflow: auto; line-height: 1.3em;");

```

Comment:
Hi @uthadehikaru - I was testing this and I like the change, but I find it even better if we make the font a bit littler, like this:

```
div.setStyle("overflow: auto; line-height: 1.3em; font-size: 12px");
```

do you like it too?

Reply:
looks good